totest-manager.py

```diff
@@ -240,26 +243,30 @@ def factory_snapshottable(self):
 
         if not self.all_repos_done('openSUSE:%s' % self.project):
             return False
+        if self.arch == "x86_64":
+            for product in ['_product:openSUSE-ftp-ftp-i586_x86_64',
```

Comment:
shouldn't this list of products come out of the ToTest class instead of depending on the arch?